main.go

```diff
@@ -347,6 +353,15 @@ func main() {
 	} else {
 		setupLog.Info("Cert-Manager is not available to the operator, skipping adding to scheme.")
 	}
+	if cfg.CollectorAvailability() == collector.Available {
+		setupLog.Info("OpenTelemetryCollectorCRDSs is available to the operator")
+	} else {
+		setupLog.Info("OpenTelemetryCollectorCRDSs are not available to the operator")
+		if !cfg.IgnoreMissingCollectorCRDs() {
+			setupLog.Error(errors.New("missing OpenTelemetryCRDs"), "The OpenTelemetryCollector CRDs are not present in the cluster. Set ignore_missing_collector_crds to true or install the CRDs in the cluster.")
```

Comment:
Should we name the specific  CRD or CRDs that are missing?

Reply:
```suggestion
			setupLog.Error(errors.New("missing OpenTelemetryCollector CRDs"), "The OpenTelemetryCollector CRDs are not present in the cluster. Set ignore_missing_collector_crds to true or install the CRDs in the cluster.")
```